ci/bamboo/build-manylinux-prototype.sh

```diff
@@ -1,101 +0,0 @@
-#!/bin/bash
```

Comment:
I think the script makes more sense in the `ci` directory. You can leave it in `ci/` if you don't want to put it in `ci/bamboo/`.